src/modules/job-manager/event.c

```diff
@@ -637,7 +637,8 @@ int event_job_update (struct job *job, json_t *event)
     }
     else if (streq (name, "exception")) {
         int severity;
-        if (job->state != FLUX_JOB_STATE_INACTIVE) {
+        if (job->state != FLUX_JOB_STATE_INACTIVE
+            && job->state != FLUX_JOB_STATE_NEW) {
```

Comment:
in the prior commit (` job-manager: relax state machine `) you changed:

```
        if (job->state == FLUX_JOB_STATE_NEW
            || job->state == FLUX_JOB_STATE_INACTIVE)
            goto inval;
```

to

```
        if (job->state != FLUX_JOB_STATE_INACTIVE) {
```

so here you add `job->state != FLUX_JOB_STATE_NEW` back.   Was there a commit series mistake somewhere in here?

Reply:
Yeah, thanks - will fix.